pkgs/development/tools/git-repo-updater/default.nix

```diff
@@ -0,0 +1,21 @@
+{ lib, python37Packages }:
+
+python37Packages.buildPythonApplication rec {
+  pname = "gitup";
+  version = "0.5";
+
+  src = python37Packages.fetchPypi {
+    inherit pname version;
+    sha256 = "11ilz9i2yxrbipyjzpfkj7drx9wkrn3phvd1a60jivphbqdldpgf";
+  };
+
+  propagatedBuildInputs = with python37Packages; [ colorama gitdb2 GitPython smmap2 ];
```

Comment:
```suggestion
  propagatedBuildInputs = [ colorama gitdb2 GitPython smmap2 ];
```

Reply:
Does it require smmap2 and gitdb2? It's not in the `install_requires` in they `setup.py`.

Reply:
Hmm, you’re right. The [Homebrew formula](https://github.com/Homebrew/homebrew-core/blob/master/Formula/gitup.rb) calls for these but the application seems to run fine without them. gitdb2 is a transitive dependency (of GitPython); I’m not sure about smmap2.